### ISSUE TITLE
Clicking "X" to delete Mashup does nothing

### ISSUE THREAD

**Issue by sstruemph** [WordImpress/Google-Maps-Builder#207]
Tested locally with all plugins disabled except Maps Builder Pro.

![screenshot-www wordimpress-mapsbuilder-new dev 2016-07-14 13-41-54](https://cloud.githubusercontent.com/assets/3521793/16851960/7d3ad7e6-49cb-11e6-90ab-12cdcaaa3d78.png)

**Comment by DevinWalker** [comment]
There will always be a single mashup item in the group - perhaps we should remove the "X" when there is a single mashup. @sstruemph can you expand on why you were trying to remove the single mashup?

**Comment by sstruemph** [comment]
Oops good point @DevinWalker . My question once mashups have been added, how can they be removed? There's a "Remove" button in the Mashup settings but it's greyed out.

**Comment by DevinWalker** [comment]
Good point @sstruemph there needs to be a better way to remove mashups.

Right now you'd have to just select a CPT that is NOT a mashup and then click "Load Markers" - it will then remove all the current mashup markers. This is confusing and hardly ideal.

**Comment by mathetos** [comment]
@DevinWalker I'd suggest a button like this:

![remove-all-mashups-button](https://cloud.githubusercontent.com/assets/930724/17334436/22152dde-588b-11e6-94e3-1958b5f9eaf3.jpg)

And clicking that would automate the process you describe above. 

I'd also suggest that the "Remove All Mashups" button only appears if a Mashup exists. And that after the Mashups are cleared out, that the remaining mashup metabox have "inactive" appended to its label so the user knows that there is actually no Mashups on the map currently.

![remove-all-mashups-button-inactive](https://cloud.githubusercontent.com/assets/930724/17334558/8249e3ca-588b-11e6-8b84-e09323e21c21.jpg)

**Comment by DevinWalker** [comment]
@mathetos nice mockups! I agree. 

1. When no mashups are on the map, no empty group should appear 
2. When you click "Remove All Mashups" on an existing mashup group, the group is removed.
3. When only 1 mashup group present and is removed that group is removed and the button is deactivated

**Comment by nishitlangaliya** [comment]
@DevinWalker ,

I have some confusion on this issue. So I have prepared my query video so can you please review video when you get a chance and let me know your feedback.

Video:
https://www.useloom.com/share/6e12358d7ac74025b44f71d735651ea2

Thanks
